### DISCUSSION TITLE
ImageUpdateAutomation - Is it possible to update other fields with the image tag?

### DISCUSSION THREAD

**Discussion by chrisjohnson00** [fluxcd/flux2#1169] [Proposals]
Loving FLUX!!
I'm using Loki for log aggregation and one nice thing I can do with it is query by pod labels, for example `app` and `version` are super helpful when it comes to non-prod CD and troubleshooting.
Is it possible to setup an `ImageUpdateAutomation` which allows setting JUST the tag version w/o the image name?
If so, that would allow me to use a policy to specify the application version/tag value in the deployment metadata labels...and include these details in my logging queries to see what container tag introduced a new error.

For example:
```
  labels:
    app: my-app
    version: v1.1.0
```

**Comment by stefanprodan** [comment]
Yes you can patch any field in any Kubernetes resource that contains either the full image URI or just the tag. Docs here https://toolkit.fluxcd.io/guides/image-update/#configure-image-update-for-custom-resources

**Reply by chrisjohnson00** [reply]
Perfect, thank you!  Works great!
